bundles/org.openhab.binding.opensprinkler/pom.xml

```diff
@@ -13,5 +13,12 @@
   <artifactId>org.openhab.binding.opensprinkler</artifactId>
 
   <name>openHAB Add-ons :: Bundles :: OpenSprinkler Binding</name>
-
+  <dependencies>
+    <dependency>
+      <groupId>javax.measure</groupId>
+      <artifactId>unit-api</artifactId>
+      <version>2.2</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
```

Comment:
Why is it necessary to add this dependency ? It is provided by default ?

Reply:
I was getting errors when trying to build, but I just rebased and removed this and all works without this now, so have it removed. Marking as solved.